website/web/templates/capture.html

```diff
@@ -67,6 +67,13 @@
         </div>
       </div>
 
+      <div class="form-group row">
+        <label for="proxy" class="col-sm-2 col-form-label">Proxy (Optional):</label>
+        <div class="col-sm-10">
+          <input type="text" class="form-control" name="proxy" id="proxy" placeholder="Pass a proxy (WIP)">
```

Comment:
Can you change the placeholder content to a description of the expected format `[scheme]://[username]:[password]@[hostname]:[port]`

And afaik, the only schemes for proxies are http/https/socks5, so we should also validate that.

Reply:
Got it.
Yes, these are the only proxy schemes supported by splash anyways.